### ISSUE TITLE
Bazel CI: BazelBuildEventServiceModuleTest  is flaky in postsubmit

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/bazel#14396]
First failure: https://buildkite.com/bazel/bazel-bazel/builds/18079
```
There were 2 failures:
1) testKeywords(com.google.devtools.build.lib.buildeventservice.BazelBuildEventServiceModuleTest)
java.lang.InterruptedException
	at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor.awaitTermination(AbstractQueueVisitor.java:532)
	at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor.awaitQuiescence(AbstractQueueVisitor.java:258)
	at com.google.devtools.build.skyframe.InvalidatingNodeVisitor.runInternal(InvalidatingNodeVisitor.java:154)
	at com.google.devtools.build.skyframe.InvalidatingNodeVisitor.run(InvalidatingNodeVisitor.java:136)
	at com.google.devtools.build.skyframe.EagerInvalidator.invalidate(EagerInvalidator.java:88)
	at com.google.devtools.build.skyframe.InMemoryMemoizingEvaluator.performInvalidation(InMemoryMemoizingEvaluator.java:290)
	at com.google.devtools.build.skyframe.InMemoryMemoizingEvaluator.evaluate(InMemoryMemoizingEvaluator.java:186)
	at com.google.devtools.build.skyframe.SequentialBuildDriver.evaluate(SequentialBuildDriver.java:38)
	at com.google.devtools.build.lib.skyframe.SequencedSkyframeExecutor.evaluateSingleValue(SequencedSkyframeExecutor.java:1021)
	at com.google.devtools.build.lib.skyframe.SequencedSkyframeExecutor.refreshWorkspaceHeader(SequencedSkyframeExecutor.java:963)
	at com.google.devtools.build.lib.skyframe.SequencedSkyframeExecutor.handleDiffs(SequencedSkyframeExecutor.java:348)
	at com.google.devtools.build.lib.skyframe.SequencedSkyframeExecutor.sync(SequencedSkyframeExecutor.java:276)
	at com.google.devtools.build.lib.runtime.CommandEnvironment.syncPackageLoading(CommandEnvironment.java:700)
	at com.google.devtools.build.lib.buildtool.util.BlazeRuntimeWrapper.executeBuild(BlazeRuntimeWrapper.java:360)
	at com.google.devtools.build.lib.buildtool.util.BuildIntegrationTestCase.buildTarget(BuildIntegrationTestCase.java:686)
	at com.google.devtools.build.lib.buildeventservice.BazelBuildEventServiceModuleTest.runBuildWithOptions(BazelBuildEventServiceModuleTest.java:161)
	at com.google.devtools.build.lib.buildeventservice.BazelBuildEventServiceModuleTest.testKeywords(BazelBuildEventServiceModuleTest.java:569)
2) testKeywords(com.google.devtools.build.lib.buildeventservice.BazelBuildEventServiceModuleTest)
io.grpc.StatusRuntimeException: CANCELLED: call already cancelled. Use ServerCallStreamObserver.setOnCancelHandler() to disable this exception
	at io.grpc.Status.asRuntimeException(Status.java:526)
	at io.grpc.stub.ServerCalls$ServerCallStreamObserverImpl.onNext(ServerCalls.java:351)
	at com.google.devtools.build.lib.buildeventservice.BazelBuildEventServiceModuleTest$DelayingPublishBuildStreamObserver$AckingThread.run(BazelBuildEventServiceModuleTest.java:872)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by meteorcloudy** [comment]
@janakdr Got it, I did a git bisect on this, and it points to https://github.com/bazelbuild/bazel/commit/882321bd2cd126e586cb555e7f4ae535d965ec3d

**Comment by janakdr** [comment]
That CL improves testing infrastructure. Looks like the test was previously throwing an exception that wasn't being processed properly?

**Comment by alexjski** [comment]
Fixed by 54e04d30b832336438324a34db8b0f6c54247b3f.
